src/MSBuild/Resources/Strings.resx

```diff
@@ -929,6 +929,120 @@
+                     (Short form: -fa)
+    </value>
+    <comment>
+      LOCALIZATION: "-featureavailability" and "-fa" should not be localized.
+      LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
+    </comment>
+  </data>
+  <data name="HelpMessage_47_TerminalLoggerSwitch" Visibility="Public">
+    <value>  -terminallogger
+                      Users are able to opt-in and use the TerminalLogger,
+                      EXCEPT when:
+                      - The terminal does not support ANSI codes or color
+                      - Output is redirected to a file or pipe
+                      (Short form: -tl)
+    </value>
+    <comment>
+      LOCALIZATION: "-terminallogger" and "-tl" should not be localized.
+      LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
+    </comment>
+  </data>
+  <data name="HelpMessage_51_GetResultOutputFileSwitch" Visibility="Public">
+    <value>  -getResultOutputFile:file
+                     Redirect output from get* into a file.
+
+                     Example:
+                     -getProperty:Bar -getResultOutputFile:Biz.txt
+                     This writes the value of property Bar into Biz.txt.
+    </value>
+    <comment>
+      LOCALIZATION: "-getResultOutputFile" should not be localized.
+      LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
+    </comment>
+  </data>
+  <data name="HelpMessage_48_TerminalLoggerParametersSwitch" Visibility="Public">
+    <value>  -termialLoggerParameters:default=[auto|false|true]
+                      Users are able to opt-in and use the TerminalLogger.
+                      - `true` forces TerminalLogger to be used even wwhen it
+                      would be disabled.
+                      - `false` forces TerminalLogger to not be used even when
+                      it would be enabled.
+                      - `auto` enables TerminalLogger when the terminal supports
+                      it and the session doesn't have redirected stdout/stderr
+                      (Short form: -tlp)
+                      Example:
+                        -termialLoggerParameters:default=auto
+    </value>
+    <comment>
+      LOCALIZATION: "-termialLoggerParameters"  and "-tlp" should not be localized.
+      LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
+    </comment>
+  </data>
+  <data name="HelpMessage_49_WaitForDebuggerSwitch" Visibility="Public">
+    <value>  -waitfordebugger
+                      Wait for debugger command line switch in Debug build.
+                      (Short form: -wfd)
+    </value>
+    <comment>
+      LOCALIZATION: "-waitfordebugger" and "-wfd" should not be localized.
+      LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.
+    </comment>
+  </data>
```

Comment:
Do not doc, unavailable in release/public builds.

Reply:
remvoe waitfordebugger help message